erpnext/hr/doctype/leave_policy_assignment/leave_policy_assignment.py

```diff
@@ -17,24 +17,31 @@ def validate(self):
 		self.validate_policy_assignment_overlap()
 		self.set_dates()
 
+	def on_submit(self):
+		self.grant_leave_alloc_for_employee()
+
+	def get_allocation(self):
+		return frappe.get_all("Leave Allocation", filters = {"leave_policy_assignment": self.name})
```

Comment:
Looks like this function is not being used anywhere. Remove it?

Reply:
done